style.css

```diff
@@ -0,0 +1,3 @@
+body {
+  background-color: darkred;
+}
```

Comment:
- [ ] This ia s testing project you don't need a style.css file. Please remove it.